build/KoreBuild.ps1

```diff
@@ -123,4 +123,20 @@ if (!(Test-Path $msbuildArtifactsDir))
 $msBuildArguments | Out-File -Encoding ASCII -FilePath $msBuildResponseFile
 
 __exec dotnet restore /p:PreflightRestore=true "$makeFileProj"
-__exec dotnet msbuild `@"$msBuildResponseFile"
+
+try {
+    __exec dotnet msbuild `@"$msBuildResponseFile"
+} finally {
+    # If the structured logger is enabled, replay the binary log to a structured log and remove the binary log
+    # (can't do this in MSBuild because MSBuild logs the log, for obvious reasons)
+    if ($env:KOREBUILD_WRITE_STRUCTURED_LOG -eq "true") {
+        # Open to suggestions for a cleaner way to do this...
+        $r = [regex]"!StructuredLoggerVersion:([^!]*)!"
+        $m = $r.Match((dotnet build $makeFileProj /t:GetStructuredLoggerVersion /p:DisableDefaultTargets=true))
+        $StructuredLoggerPath = "$env:USERPROFILE\.nuget\packages\microsoft.build.logging.structuredlogger\$($m.Groups[1].Value)\lib\netstandard1.5\StructuredLogger.dll"
```

Comment:
Is this path problematic? like in our CI we have the package folder in the build agent working dir...also could you update the travis and appveyor xml files for this PR so that we can take a look at how this works on them (and hopefully get the artifacts)